src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php

```diff
@@ -397,18 +398,23 @@ protected function isCsrfTokenValid(string $id, ?string $token): bool
     /**
      * Dispatches a message to the bus.
      *
-     * @param object|Envelope $message The message or the message pre-wrapped in an envelope
+     * @param object|Envelope  $message The message or the message pre-wrapped in an envelope
+     * @param StampInterface[] $stamps
      *
      * @final
      */
-    protected function dispatchMessage($message): Envelope
+    protected function dispatchMessage($message/*, array $stamps = []*/): Envelope
```

Comment:
As the method is marked as being `@final`, we don't need the BC break code. We can add the new argument directly.

Reply:
@fabpot BC break code removed.